node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs');
+
+const argv = process.argv;
+
+fs.readFile(argv[2], (err, data) => {
```

Comment:
There should be a second argument here of `utf8`, then you won't need to use `toString` on your `data`.
```JAVASCRIPT
fs.readFile(argv[2], 'utf8', (err, data) => {
```